### PR TITLE
BTS-1077: bulk test data creation

### PR DESCRIPTION
**Pull request by dothebart**
### Scope & Purpose

reduce test resource usage by properly bulking test data creation

- [x] :hankey: Bugfix

### CODE REVIEW

**Comment by dothebart** [comment]
tests green except unrelated errors.

File: tests/js/common/shell/shell-pregel-multicollection-edgecol.js [hunk 1]
```diff
@@ -122,9 +123,16 @@ function multiCollectionTestSuite() {
           vertices[i].push({ _key: String(c) + ":" + String(x++) });
         }
         for (let i = 0; i < cn; ++i) {
-          db[`${vColl}_${i}`].insert(vertices[i]);
+          let coln = `${vColl}_${i}`;
+          if (!cols.hasOwnProperty(coln)) {
+            cols[coln] = [];
+          }
+          cols[coln] = cols[coln].concat(vertices[i]);
```
Comment (jsteemann) [inline tests/js/common/shell/shell-pregel-multicollection-edgecol.js:130]:
```suggestion
          cols[coln].push(vertices[i]);
```

File: tests/js/common/shell/shell-pregel-multicollection-edgecol.js [hunk 2]
```diff
@@ -155,7 +164,11 @@ function multiCollectionTestSuite() {
         }
         for (let i = 0; i < cn; ++i) {
           for (let j = 0; j < cn; ++j) {
-            db[`${eColl}_${i}_${j}`].insert(edges[i][j]);
+            let coln = `${eColl}_${i}_${j}`;
+            if (!edgeCols.hasOwnProperty(coln)) {
+            edgeCols[coln] = [];
+          }
+          edgeCols[coln] = edgeCols[coln].concat(edges[i][j]);
```
Comment (jsteemann) [inline tests/js/common/shell/shell-pregel-multicollection-edgecol.js:171]:
```suggestion
            if (!edgeCols.hasOwnProperty(coln)) {
              edgeCols[coln] = [];
            }
            edgeCols[coln].push(edges[i][j]);
```

File: tests/js/common/shell/shell-pregel-multicollection-edgecol.js [hunk 3]
```diff
@@ -168,9 +181,16 @@ function multiCollectionTestSuite() {
           const vTo = (toId + c) % cn;
           const from = `${vColl}_${vFrom}/${fromKey}`;
           const to = `${vColl}_${vTo}/${toKey}`;
-          db[`${eColl}_${vFrom}_${vTo}`].insert({ _from: from, _to: to, vertex: String(fromKey) });
+          let coln = `${eColl}_${vFrom}_${vTo}`;
+          if (!cols.hasOwnProperty(coln)) {
+            cols[coln] = [];
+          }
+          edgeCols[coln] = edgeCols[coln].concat({ _from: from, _to: to, vertex: String(fromKey) });
```
Comment (jsteemann) [inline tests/js/common/shell/shell-pregel-multicollection-edgecol.js:188]:
```suggestion
          edgeCols[coln].push({ _from: from, _to: to, vertex: String(fromKey) });
```

File: tests/js/common/shell/shell-pregel-multicollection-edgecol.js [hunk 4]
```diff
@@ -155,7 +164,11 @@ function multiCollectionTestSuite() {
         }
         for (let i = 0; i < cn; ++i) {
           for (let j = 0; j < cn; ++j) {
-            db[`${eColl}_${i}_${j}`].insert(edges[i][j]);
+            let coln = `${eColl}_${i}_${j}`;
+            if (!edgeCols.hasOwnProperty(coln)) {
+            edgeCols[coln] = [];
+          }
```
Comment (jsteemann) [inline tests/js/common/shell/shell-pregel-multicollection-edgecol.js:170]:
```suggestion
            if (!edgeCols.hasOwnProperty(coln)) {
              edgeCols[coln] = [];
            }
```

**Pull request merged**